liberty-maven-webapp-parent/pom.xml

```diff
@@ -55,6 +55,13 @@
                             </configuration>
                         </execution>
                         <execution>
+                            <id>export-plugin-config</id>
+                            <phase>compile</phase>
+                            <goals>
+                                <goal>export-plugin-config</goal>
```

Comment:
This goal is to output the plugin configuration settings to an xml file as long as the plugin is included in the project.  It doesn't really matter which phase to run or what package type it is.  When we import the project to Eclipse, the project will be built automatically by default (i.e compile), so I set it to match the Maven compile phase.